src/components/form/EnzymeMultiSelect.jsx

```diff
@@ -10,7 +10,7 @@ export default function EnzymeMultiSelect({ setEnzymes }) {
   const [error, setError] = React.useState(false);
   const [waitingMessage, setWaitingMessage] = React.useState('retrieving enzyme list from server...');
   React.useEffect(() => {
-    const url = `${import.meta.env.VITE_REACT_APP_BACKEND_URL}restriction_enzyme_list`;
+    const url = `${import.meta.env.VITE_REACT_APP_BACKEND_URL}/restriction_enzyme_list`;
```

Comment:
> As a side note, do you know why having a `VITE_REACT_APP_BACKEND_URL` without a trailing slash will cause the app to fail contacting backend (no error in console)?

Yes, this was fairly stupid, I was not adding the slash between the domain name and the endpoint in the code:

Reply:
Then I'd suggest adding some code to make sure the thing work with our without end slash. Then the information about its necessity can be removed.